exporter/tanzuobservabilityexporter/metrics.go

```diff
@@ -188,6 +190,31 @@ func getValue(numberDataPoint pdata.NumberDataPoint) (float64, error) {
 	}
 }
 
+// pushGaugeSingleNumberDataPoint sends a metric as a gauge metric to tanzu
+// observability. metric is the metric to send. numberDataPoint is the value
+// of the metric. Any errors get appended to errs. sender is what sends the
+// gauge metric to tanzu observability. logger is the logger. missingValues
+// keeps track of metrics with missing values.
+func pushGaugeSingleNumberDataPoint(
```

Comment:
Consider renaming to `pushGaugeNumberDataPoint` – and if we were passing a slice of data points, make it `pushGaugeNumberDataPoints`.

Reply:
Done.